test_unstructured_ingest/expected-structured-output/onedrive/utic-test-ingest-fixtures/tests-example.xls.json

```diff
@@ -183,27 +183,6 @@
     "text": "http://www.cmu.edu/blackboard/files/evaluate/tests-example.xls",
     "type": "Title"
   },
```

Comment:
These are each a duplicate of the element above, which indicates that particular double-emit behavior is fixed.